### ISSUE TITLE
msmarco-MiniLM-L6-cos-v5 hyperparameters

### ISSUE THREAD

**Issue by vjeronymo2** [UKPLab/sentence-transformers#1220]
Hey Nils
Can you share the hyperparameters for the msmarco-MiniLM-L6-cos-v5 model and script call as well? I assume you used the train_bi-encoder_mnrl.py.
I'm trying to reproduce it but I'm not even close to the MRR@10 you got on MSMARCO of 32.27.

Thanks in advance!

**Comment by vjeronymo2** [comment]
I already ran your fine tuned msmarco-MiniLM-L6-cos-v5 and got the same MRR@10 of 32.27 =)
Training train_bi-encoder_mnrl.py with the following parameters yielded MRR@10 of 0.3040:
--lr=1e-5
--warmup_steps=10000
--num_negs_per_system=10
--epochs=20
--train_batch_size 32
--accumulation_steps 4            # gradient accumulation
--negs_to_use=distilbert-margin_mse-sym_mnrl-mean-v1

I'll try using the default parameters and this batch of 140 you specified

**Comment by vjeronymo2** [comment]
I wanna get the parameters right before I train a bi-encoder multilingual miniLM (I'm working with Rodrigo)
